documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/main.cadl

```diff
@@ -0,0 +1,60 @@
+import "@cadl-lang/rest";
+import "@cadl-lang/versioning";
+import "@azure-tools/cadl-autorest";
+import "@azure-tools/cadl-azure-core";
+import "../Contoso.WidgetManager.Shared";
+
+using Cadl.Http;
+using Cadl.Rest;
+using Cadl.Versioning;
+using Azure.Core;
+
+@service({
+    title: "Contoso Widget Manager",
+  })
+@versioned(Contoso.WidgetManager.Versions)
+@versionedDependency([[Contoso.WidgetManager.Versions.v2022_11_01_Preview, Azure.Core.Versions.v1_0_Preview_1]])
+namespace Contoso.WidgetManager;
```

Comment:
```suggestion
namespace Azure.Contoso.WidgetManager;
```

Reply:
For 1st party service, I think it starts with `Microsoft`. But you can certainly use anything and use `@armProviderNamespace` to set the Provider name used in the path.

Reply:
Use 'Azure' to be consistent with generated sdk namespace.